pkg/cmd/log.go

```diff
@@ -46,6 +46,9 @@ func newCmdLog(rootCmdOptions *RootCmdOptions) (*cobra.Command, *logCmdOptions)
 		RunE:    options.run,
 	}
 
+	cmd.Flags().BoolP("tail", "f", false, "To show the end of the logs")
+	cmd.Flags().Int64P("tail-lines", "l", 10, "The number of lines from the end of the logs to show")
```

Comment:
I noticed that `-f` shorthand generally means "follow" as in `kubectl logs -f` and our `kamel log` has this behaviour. So `-f` may not be a good shorthand character. `-t` is also not a good shorthand as it's already used in `kamel run` as `--trait`. Maybe we should simply force `--tail`.

Reply:
I agree, let's do that

Reply:
Or like `kubectl logs` we could possibly drop the original `--tail` and always force passing line numbers with `--tail` without a default value:
https://github.com/kubernetes/kubectl/blob/master/pkg/cmd/logs/logs.go

Reply:
Good idea, let's use the same behavior as `kubectl logs`:
```
    --tail=-1:
	Lines of recent log file to display. Defaults to -1 with no selector, showing all log lines otherwise 10, if a
	selector is provided.
```